lib/suse/connect/yast.rb

```diff
@@ -7,6 +7,19 @@ module Connect
     class YaST
       class << self
 
+        # Returns a hash containing default configuration values.
+        # Keys: :config_file_path, :scc_url, :server_cert_file_path, :update_certificates_script_path, :credentials_dir_path, :global_credentials_file_path
+        # @return [Hash]
+        def defaults
```

Comment:
I would suggest to redefine Connect constants in YaST wrapper (e.g. SUSE::Connect::Config::DEFAULT_CONFIG_FILE => SUSE::Connect::YaST::DEFAULT_CONFIG_FILE)

```ruby
A_CONST = 'foo'
B_CONST = A_CONST

puts A_CONST
puts B_CONST
```

1) This way the YaST team can operate on them in the same they we do in our code (e.g. const_defined?)
2) The constants can not be changed
3) YaST team should not adapt a whole source code but just replace the constant names in the code
4) And it is easy to test 

@kalabiyau, @digitaltom What do you think?